apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -165,7 +166,8 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
         state.config.ui.isClientSecretHashEnabled);
     const orgType: OrganizationType = useSelector((state: AppState) =>
         state?.organization?.organizationType);
-
+    const isFAPIAppCreationEnabled: boolean = useSelector((state: AppState) =>
+        state.config.ui.features.fapiApplicationCreation.enabled);
```

Comment:
Shall we add null checks?

```suggestion
        state.config.ui.features?.fapiApplicationCreation?.enabled);
```

Reply:
added with [db451e6](https://github.com/wso2/identity-apps/pull/4616/commits/db451e67d57f674883d760ba9b0831bb14accd8f)